contracts/truefi2/CreditModel.sol

```diff
@@ -281,7 +281,7 @@ contract CreditModel is ICreditModel, UpgradeableClaimable {
         if (stakedAmount == 0) {
             return 0;
         }
-        return pool.oracle().truToToken(stakedAmount).mul(borrowLimitConfig.ltvRatio).div(BASIS_POINTS);
+        return pool.oracle().truToUsd(stakedAmount).mul(borrowLimitConfig.ltvRatio).div(BASIS_POINTS);
```

Comment:
If we go with this implementation, perhaps we should rename this function to `conservativeCollateralValueInUsd()` to reflect the fact that it's no longer denominated in token amounts.

---

Any thoughts on how this will interact with PR #1053? If this function is denominated in 18-decimal USD, then we'll need to convert this value back to token amounts when adjusting effective credit scores. So it's either simpler borrow limits or simpler rates. Which do you think is cleaner?

I'm leaning somewhat toward keeping this as `truToToken()`:
1. we can revert the oracle changes and reduce our upgrade burden for deployment
2. `poolBorrowMax()` already does decimal conversions for `poolValue()` so it wouldn't be strange for it to also handle them for `conservativeCollateralValue()`.

Any other tradeoffs I'm missing?